mlflow/tracing/utils/otlp.py

```diff
@@ -0,0 +1,61 @@
+import os
+from typing import Optional
+
+from opentelemetry.sdk.trace.export import SpanExporter
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import RESOURCE_DOES_NOT_EXIST
+
+
+def should_use_otlp_exporter() -> bool:
+    return _get_otlp_endpoint() is not None
+
+
+def get_otlp_exporter() -> SpanExporter:
+    """
+    Get the OTLP exporter based on the configured protocol.
+    """
+    endpoint, protocol = _get_otlp_endpoint(), _get_otlp_protocol()
```

Comment:
```suggestion
    endpoint = _get_otlp_endpoint()
    protocol = _get_otlp_protocol()
```

Can we split this line?